presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1127,7 +1127,7 @@ public SystemSessionProperties(
                 booleanProperty(
                         ENABLE_DISTRIBUTED_TRACING,
                         "Enable distributed tracing of the query",
-                        tracingConfig.getEnableDistributedTracing(),
+                        false,
```

Comment:
Why this change?

Reply:
Because the system set session here does not really make sense. The session property set by users is used at the very beginning at the post statement, where even set by default will not be seen at that stage. So if the default of the enable_distributed_tracing is true here, the system will still not trace at the post stage unless the user client explicitly set the session to be true. 
This will bring up a situation where users check session properties and sees this one default to true, and start a query but the query is not actually traced.

I used this tracing config property at system startup to determine which tracer to use based on if tracing is enabled.

I am not sure if I explained it clearly but let me know and we can VC to clarify.

Reply:
```java
SessionContext sessionContext = new HttpRequestSessionContext(servletRequest, sqlParserOptions, tracer, sessionProperties);